vault/instances.tf

```diff
@@ -3,7 +3,7 @@ locals {
 }
 
 module "vault1" {
-  source        = "github.com/skyscrapers/terraform-instances//instance?ref=2.0.15"
+  source        = "../instance"
```

Comment:
Hmm, you've got a point there. We do this in other places of the module too though.

For Vault it doesn't matter too much, the instances can be destroyed/recreated without problem, since no state is stored.

Reply:
there is state stored though, the unseal keys.